experimental/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to experimental packages in this project will be documented
 
 * feat(api-logs): Add delegating no-op logger provider [#4861](https://github.com/open-telemetry/opentelemetry-js/pull/4861) @hectorhdzg
 * feat(instrumentation-http): Add support for client span semantic conventions 1.27 [#4940](https://github.com/open-telemetry/opentelemetry-js/pull/4940) @dyladan
+* feat(instrumentation-http): Add support for server span semantic conventions 1.27 [#4978](https://github.com/open-telemetry/opentelemetry-js/pull/4978) @dyladan
```

Comment:
I think we could have a more detailed changelog in here for releasing. Something splashier will get the attention of folks looking for this, i.e. the note in the README "`OTEL_SEMCONV_STABILITY_OPT_IN` environment variable can now be set to `http` or `http/dup` to emit stable attributes. See README for details." or something like that? I'll add a follow-up note to the tracking issue too that we'll want to update docs (looks like some folks are already doing some of that, but a pass-through for missing updates will be good)

Reply:
I'll update it